src/storage/rpc.rs

```diff
@@ -33,7 +33,8 @@ pub struct RpcStorage {
     // back to our `Storage` trait.
     // Not using `AHashMap` for ease of serialization.
     // TODO: Cache & snapshot block hashes too!
```

Comment:
Can remove now!

Reply:
> // Not using `AHashMap` for ease of serialization.

Actually, since we are customizing the serialization, let's use `AHashMap`, then format the output later.